backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -319,7 +354,7 @@ private CanvasAPIException throwCanvasException(Exception e) {
      * @return JsonNode
      * @throws IOException error message thrown
      */
-    protected JsonNode parseResponseToJsonNode(Response response) throws IOException {
+    public static JsonNode parseResponseToJsonNode(Response response) throws IOException {
```

Comment:
Why did you move this to public? We made it protected since it is a helper function that shouldn't be accessible to other classes.